cmd/util/ledger/migrations/storage_v5.go

```diff
@@ -160,7 +160,7 @@ func (m *StorageFormatV5Migration) Migrate(payloads []ledger.Payload) ([]ledger.
 	return migratedPayloads, nil
 }
 
-func (m StorageFormatV5Migration) getContractsOnlyAccounts(payloads []ledger.Payload) *state.Accounts {
+func (m StorageFormatV5Migration) getContractsOnlyAccounts(payloads []ledger.Payload) *state.AccountsState {
```

Comment:
This return type could be `state.Accounts` which would mean `AccountsState` wouldn't need to be exported from the fvm.